lib/ansible/modules/windows/win_zip.ps1

```diff
@@ -0,0 +1,253 @@
+#!powershell
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+#Requires -Version 5.0
+
+Set-StrictMode -Version 2.0
+
+$ErrorActionPreference = "Stop"
```

Comment:
Doesn't need to be set in the module, this is set automatically by the module runner.